scrapy/utils/reqser.py

```
@@ -47,7 +50,11 @@ def request_from_dict(d, spider=None):
     eb = d['errback']
     if eb and spider:
         eb = _get_method(spider, eb)
-    return Request(
+    try:
```

I think in this case it is better to fail loudly with an exception.